lib/manager/terragrunt-version/extract.ts

```diff
@@ -0,0 +1,13 @@
+import * as datasourceGitHubRelease from '../../datasource/github-releases';
+import { logger } from '../../logger';
+import { PackageDependency, PackageFile } from '../common';
+
+export function extractPackageFile(content: string): PackageFile {
+  logger.trace('terragrunt-version.extractPackageFile()');
+  const dep: PackageDependency = {
+    depName: 'gruntwork-io/terragrunt',
+    currentValue: content.trim(),
+    datasource: datasourceGitHubRelease.id,
```

Comment:
I guess terragrunt uses terraform versioning?, so should be added here too.

Reply:
Not sure I understand.

terragrunt uses terraform versioning in what way?

Reply:
Looks like we need a custom versioning here

https://github.com/cunymatthieu/tgenv#terragrunt-version

can be a version, `latest` or `latest:<regex>` so we should at least ignore the latest values with this pr, otherwise renovate would try to update them to real versions.

like here:
https://github.com/renovatebot/renovate/blob/9a1c4fabb74c2e943a346d48049d7be1d86966da/lib/manager/gomod/extract.ts#L24-L26

Reply:
Firstly, thanks for all the feedback!

As you've likely figured out by now I'm not familiar with this codebase or typescript 😅 

So this PR may be open for a while as a figure it out 🙃 

Can I make the case for not including this right now?

- tfenv has the same versioning: https://github.com/tfutils/tfenv#terraform-version-file
   - Yet that's not accounted for in the `terraform-version` manager in renovate
- I'd argue the main use case of these version managers tools and renovate are precise version bumps

Reply:
so `terraform-version` manager is wrong too 🙈

Reply:
The lookup process will already do a filter based on `isValid()`. We prefer not to set unsupported version in manager extracts so that users can configure a custom versioning using package rules that gets applied post-extract and pre-validation